packages/modules/src/modules/banners/charityAppeal/CharityAppealBanner.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import { CharityAppealBannerUnvalidated as CharityAppealBanner } from './CharityAppealBanner';
+import { props, contentCharityAppeal } from '../utils/storybook';
+import { BannerProps, SecondaryCtaType } from '@sdc/shared/types';
+
+export default {
+    component: CharityAppealBanner,
+    title: 'Banners/CharityAppeal',
+    args: props,
+} as Meta;
+
+const Template: Story<BannerProps> = (props: BannerProps) => <CharityAppealBanner {...props} />;
+
+export const Default = Template.bind({});
+Default.args = {
+    content: {
+        ...contentCharityAppeal,
+    },
+};
+
+export const WithReminder = Template.bind({});
+WithReminder.args = {
+    content: {
+        ...contentCharityAppeal,
+        secondaryCta: {
+            type: SecondaryCtaType.ContributionsReminder,
+        },
+    },
+};
```

Comment:
The `WithReminder` story here is broken, just having a look into it but it seems that this may have also been the case for the Contrib banner this was modelled off of

Reply:
Caught up on this one